tests/testthat/_snaps/template.md

```diff
@@ -84,35 +80,30 @@
 
     Code
       prob_metric_summarizer(name = "roc_auc", fn = roc_auc_vec, data = hpc_f1,
-        truth = obs, estimate = matrix(data = c(HELLO = HELLO, F = F, M = M, L = L),
-        ncol = 4L))
+        truth = obs, c(HELLO, F, M, L))
     Condition
-      Error in `dplyr::summarise()`:
-      ! Problem while computing `.estimate = fn(...)`.
-      Caused by error in `matrix()`:
-      ! object 'HELLO' not found
+      Error:
+      ! Can't subset columns that don't exist.
+      x Column `HELLO` doesn't exist.
 
 ---
 
     Code
       prob_metric_summarizer(name = "roc_auc", fn = roc_auc_vec, data = hpc_f1,
-        truth = obviouslywrong, estimate = matrix(data = c(VF = VF, F = F, M = M, L = L),
-        ncol = 4L))
+        truth = obviouslywrong, VF:L)
     Condition
-      Error in `dplyr::summarise()`:
-      ! Problem while computing `.estimator = finalize_estimator(obviouslywrong, estimator, name)`.
-      Caused by error:
-      ! object 'obviouslywrong' not found
+      Error:
+      ! Can't subset columns that don't exist.
+      x Column `obviouslywrong` doesn't exist.
 
 ---
 
     Code
       prob_metric_summarizer(name = "roc_auc", fn = roc_auc_vec, data = hpc_f1,
-        truth = obs, estimate = matrix(data = c(VF = VF, F = F, M = M, L = L), ncol = 4L),
-        obviouslywrong = TRUE)
+        truth = obs, VF:L, obviouslywrong = TRUE)
     Condition
-      Error in `prob_metric_summarizer()`:
-      ! `...` must be empty.
-      x Problematic argument:
-      * obviouslywrong = TRUE
+      Error:
+      ! Must subset columns with a valid subscript vector.
+      x Subscript `TRUE` has the wrong type `logical`.
+      i It must be numeric or character.
```

Comment:
Weird case where `obviouslywrong = TRUE` actually gets passed through the `...` to the tidyselect function, but I'm not worried about this